data/macros.meson

```diff
@@ -2,6 +2,12 @@
 %__meson_wrap_mode nodownload
 %__meson_auto_features enabled
 
+%_smp_mesonflags %([ -z "$RPM_BUILD_NCPUS" ] \\\
```

Comment:
this definitely should not be done this way... you need to use new RPM flag which specifies how many CPUs you have.

Reply:
Ah, whoops. I meant to change that and forgot. Will push a fix momentarily.

Reply:
Wait, when you say "new RPM flag", do you mean that RPM itself will report the number of CPUs and we can avoid `getconf`?